### DISCUSSION TITLE
How to get round number for current season

### DISCUSSION THREAD

**Discussion by dalton-f** [jolpica/jolpica-f1#93] [Q&A] [answered]
Is there an easy way to find out the current round number for the current season? It seems possible if you use the results endpoint to get the last recorded result and grab the round number from that, but I was wondering if theres an easier way to fetch it?

**Comment by jolpica** [comment, marked as answer]
I think the best way currently would be using the races endpoint and getting the value from the race table.
https://api.jolpi.ca/ergast/f1/current/last/

This is a bit limited at the moment as I believe we don't switch the `last` / `next` round until the day after the race. Its something we hope to improve in the future.

**Reply by dalton-f** [reply]
Thanks for the quick response! I'll make use of that endpoint instead haha. No worries about the limitations, I want to keep the API calls to a minimum anyway.

**Reply by dalton-f** [reply]
On an extension to this question, is there an easy way to determine how much sprints are left in the season?

**Reply by jolpica** [reply]
Nothing to give you a number directly unfortunately, but this endpoint would give you all the sprint races in a season, which you could combine with a date check or a round number to see how many are left:
https://api.jolpi.ca/ergast/f1/2024/sprint/
